packages/dashboards/tests/integration/components/navi-dashboard-test.js

```diff
@@ -115,4 +116,25 @@ test('dashboard export', function(assert) {
 
     config.navi.FEATURES.enableDashboardExport = originalFeatureFlag;
   });
+})
+
+
+test('dashboard schedule', function (assert) {
+  assert.expect(2);
+  let originalFeatureFlag = config.navi.FEATURES.enableScheduleDashboards;
+
+  return wait().then(() => {
+    config.navi.FEATURES.enableScheduleDashboards = true;
+    this.render(hbs`{{navi-dashboard dashboard=dashboardModel}}`);
+
+    assert.ok($('.action.schedule').is(':visible'),
+      'Dashboard schedule button should be visible');
+
+    config.navi.FEATURES.enableScheduleDashboards = false;
+    this.render(hbs`{{navi-dashboard dashboard=dashboardModel}}`);
+    assert.notOk($('.action.schedule').is(':visible'),
+      'Dashboard schedule button should not be visible');
+
+    config.navi.FEATURES.enableScheduleDashboards = originalFeatureFlag;
+  });
```

Comment:
Can we test a few other things:

- after clicking the `schedule` button, the model opens
- verify you can't schedule a dashboard that isn't owned by the user

Reply:
> after clicking the schedule button, the model opens
There is already test for this

> verify you can't schedule a dashboard that isn't owned by the user
 The button will not been shown to other users